modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -45,58 +45,33 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state, array $config = NULL): array {
+    if (empty($config['input']['name'])) {
+      return [];
+    }
+
     $form_state->set('oe_whitelabel_search_config', $config);
-    $input_value = '';
 
-    if (!empty($config['input']['name'])) {
-      $input_value = $this->getRequest()->get($config['input']['name']);
-    }
+    $form['#region'] = $config['form']['region'];
 
     $form['search_input'] = [
-      '#prefix' => '<div class="bcl-search-form__group w-100">',
-      '#suffix' => '</div>',
       '#type' => 'textfield',
       '#title' => $config['input']['label'],
       '#title_display' => 'invisible',
       '#size' => 20,
-      '#margin_class' => 'mb-0',
+      '#form_id' => $this->getFormId(),
+      '#region' => $config['form']['region'],
```

Comment:
I know we are adding these values for the theme hook suggestions.
If we follow my proposal we no longer need it, see https://github.com/openeuropa/oe_whitelabel/commit/2a5695e4b6cf4c3ff132ff1cb0b0a005afec7f4e